liberty-maven-plugin/pom.xml

```diff
@@ -84,7 +84,7 @@
         <dependency>
             <groupId>io.openliberty.tools</groupId>
             <artifactId>ci.common</artifactId>
-            <version>1.8.11</version>
+            <version>1.8.12-SNAPSHOT</version>
```

Comment:
did you want to commit the updated ci.common version in this PR?

Reply:
yes